libs/components/tabs/testing/src/modules/sectioned-form/sectioned-form-section-content-harness.ts

```diff
@@ -0,0 +1,40 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+
+import { SkyVerticalTabContentHarness } from '../vertical-tabset/vertical-tab-content-harness';
+
+import { SkySectionedFormSectionContentHarnessFilters } from './sectioned-form-section-content-harness-filters';
+
+/**
+ * Harness for interacting with a vertical tab content in tests.
+ */
+export class SkySectionedFormSectionContentHarness extends SkyVerticalTabContentHarness {
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkySectionedFormSectionContentHarness` that meets certain criteria.
+   */
+  public static override with(
+    filters: SkySectionedFormSectionContentHarnessFilters,
+  ): HarnessPredicate<SkySectionedFormSectionContentHarness> {
+    return SkySectionedFormSectionContentHarness.getDataSkyIdPredicate(
```

Comment:
would a `super.with(filters) as SkySectionedFormSectionContentHarness` or something similar not work here instead of rewriting the `addOption` code?

Reply:
changed